storage/internal/benchmarks/main.go

```diff
@@ -295,40 +295,30 @@ type benchmarkResult struct {
 	endMem        runtime.MemStats
 }
 
-func (br *benchmarkResult) selectParams(opts benchmarkOptions) {
-	api := opts.api
-	if api == mixedAPIs {
-		if randomBool() {
-			api = xmlAPI
-		} else {
-			api = grpcAPI
-		}
+func (br *benchmarkResult) selectReadParams(opts benchmarkOptions, api benchmarkAPI) {
+	if api == jsonAPI {
```

Comment:
This is part of what I want to re-factor to be less confusing. For now, it's like this since json is not a valid API for downloads, but this would be changed in the JSON PR and we no longer do this there.

Reply:
Ahh gotcha, I think I'm not totally clear on how these stack. Would you merge that one first? I can approve this sooner if you just need to get this one in...

Reply:
I don't mind whichever way they merge in(either way there'll be conflicts), but this one should get in as soon as possible